src/main/java/aquality/appium/mobile/elements/Element.java

```diff
@@ -93,4 +84,16 @@ public void sendKeys(Keys key) {
     public IElementTouchActions getTouchActions() {
         return new ElementTouchActions(this);
     }
+
+    @Override
+    public Point getCenter() {
+        logElementAction("loc.get.center");
+        WebElement element = getElement();
+        Point upperLeft = element.getLocation();
+        Dimension dimensions = element.getSize();
+        Point center = new Point(upperLeft.getX() + dimensions.getWidth() / 2,
+                upperLeft.getY() + dimensions.getHeight() / 2);
+        logElementAction("loc.center.value", center.getX(), center.getY());
```

Comment:
do we need to log element name as well?

Reply:
we already do this
![image](https://user-images.githubusercontent.com/9014926/154054392-84fb3c5a-4ff4-4aa9-9c81-fb720c627bad.png)
2022-02-14 19:05:16 INFO  - Button '1' :: Getting center coordinates of the element
2022-02-14 19:05:16 INFO  - Button '1' :: Center coordinates of the element: (x:123; y:416)